src/test/resources/com/scriptbasic/testprograms/TestSelect3.bas

```diff
@@ -0,0 +1,29 @@
+'
+' This program is used in unit testing the jScriptBasic interpreter to test the functionality 
```

Comment:
Just my 2c: this is not a unit test. It is an integration test. jScriptBasic has very few unit tests. It mainly has integration tests that run under the JUnit framework.

Reply:
I've used comment from TestIf.bas :-)
Resolved.